deploy/verify_polyscan.js

```diff
@@ -0,0 +1,23 @@
+const { ethers } = require('hardhat');
+require('dotenv').config({ path: '.env' });
```

Comment:
```suggestion
```
As this is supposed to be run within the hardhat environment, I think this is not needed